openquakeplatform/openquakeplatform/management/commands/updatures/classes.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+
+import json
+import os
+import sys
+import copy
+from pprint import pprint
+from collections import OrderedDict
+from django.core.management import call_command, execute_manager
+
+# from updatures import debug_level
+import pdb
+
+class backinheritance(object):
```

Comment:
CamelCase done, about the class without methods: I prefer keep BackInheritance as class because in the future some extension could need methods for this class.